agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -94,6 +97,15 @@ private List<TelemetryItem> convertOtelMetricToAzureMonitorMetric(
       MetricData metricData, boolean isPreAggregatedStandardMetric) {
     List<TelemetryItem> telemetryItems = new ArrayList<>();
 
+    String namespace = null;
+    if (micrometerMetricNamespace != null
+        && metricData
```

Comment:
```
metricData
            .getInstrumentationScopeInfo()
            .getName()
            .equals("io.opentelemetry.micrometer-1.5")
```

Something shorter to have something easier to read? For example, with an extract local variable (as boolean isMicrometer1_5) or an extract method.

Reply:
done